opencolorio_config_aces/clf/discover/classify.py

```diff
@@ -1157,18 +1204,21 @@ def classify_clf_transforms(unclassified_clf_transforms):
     --------
     >>> clf_transforms = classify_clf_transforms(
     ...     discover_clf_transforms())
-    >>> family = sorted(clf_transforms.keys())[1]
+    >>> family = sorted(clf_transforms.keys())[0]
     >>> str(family)
-    'Utility'
+    'blackmagic'
```

Comment:
Should vendor names be capitalized for family? `Utility` is `utility` in the directory structure, so the `blackmagic` directory should probably be `Blackmagic` for family.

Reply:
Yes, so we could do that and it would be fine, the code is actually using the parsed `CLFtransformID` for the config colorspaces which is a bit more solid as the name are expected to be good there! We could make them match that being said.

Shall we do that post Siggraph? There are always shenanigans when trying to change case of files and directories with Git.

Reply:
Sounds good. Post Siggraph is fine.